pickup/lib/consumer.js

```diff
@@ -66,7 +66,9 @@ export async function createConsumer ({
   dynamoTable,
   dynamoEndpoint,
   downloadStatusManager,
-  downloadStatusLoggerSeconds = 300 // logs every 5 minutes
+  downloadStatusLoggerSeconds = 300, // logs every 5 minutes
+  shutdownOnIdleAfterSecond = 0, // 0 no shutdown, > 0 shutdown after no messages are received
```

Comment:
Pluralise seconds to match `SHUTDOWN_ON_IDLE_AFTER_SECONDS` and `downloadStatusLoggerSeconds`
```suggestion
  shutdownOnIdleAfterSeconds = 0, // 0 no shutdown, > 0 shutdown after no messages are received
```